src/shared/seccomp-util.c

```diff
@@ -1112,6 +1112,17 @@ int seccomp_memory_deny_write_execute(void) {
                 r = seccomp_rule_add_exact(
                                 seccomp,
                                 SCMP_ACT_ERRNO(EPERM),
+                                SCMP_SYS(mmap2),
```

Comment:
I don't think this is necessary. mmap2() is just an minor implementation difference on one arch. The concept is clear, and the call is exposed in glibc as mmap() anyway...